unitTests/ui/FacetTest.ts

```diff
@@ -183,6 +182,43 @@ export function FacetTest() {
       );
     });
 
+    it('should log an analytics event when showing more results', done => {
+      const results = FakeResults.createFakeResults(test.cmp.options.pageSize + 5);
+      const validation = Promise.resolve(results.results);
+      const spyFacetQueryController = jasmine.createSpy('spyFacetQueryController').and.returnValue(validation);
+      const expectedMetadata = jasmine.objectContaining({
+        facetId: test.cmp.options.id,
+        facetField: test.cmp.options.field.toString(),
+        facetTitle: test.cmp.options.title
+      });
+
+      test.cmp.facetQueryController.fetchMore = spyFacetQueryController as any;
+      test.cmp.showMore();
+      validation.then(() => {
```

Comment:
Could use async here for better readability.

Change the function to `async done => { .... }`

And do: 
```
test.cmp.showMore();
await validation;
expect(.....);
```